build/automake/build.sh

```diff
@@ -28,12 +28,15 @@
 . ../../lib/functions.sh
 
 PROG=automake
-VER=1.15
+VER=1.15.1
 VERHUMAN=$VER
 PKG=developer/build/automake
 SUMMARY="GNU Automake $VER"
 DESC="GNU Automake - A Makefile generator ($VER)"
 
+# make sure test uses gnu bins
+PATH=/usr/gnu/bin:$PATH
+
```

Comment:
Add:
```
[ -n "$BATCH" ] && SKIP_TESTSUITE=1
```

since the test-suite takes over an hour to run, we don't want to run it during a full end-to-end build (batch mode), just when testing package upgrades.